index.html

```diff
@@ -2,188 +2,242 @@
+                        <li>CSS</li>
                         <li>HTML</li>
                         <li>Bootstrap</li>
                         <li>Ruby</li>
                     </ul>
+                    <button class="btn">See Project</button>
                 </div>
-                <button>See project</button>
             </div>
 
-            <div class="card">
-                <div class="card-content">
-                    <h2>Profesional Art Printing Data</h2>
-                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the
-                        industry's standard.</p>
-                    <ul>
-                        <li>HTML</li>
-                        <li>Bootstrap</li>
-                        <li>Ruby</li>
-                    </ul>
+            <div class="my-cards">
+                <div class="card">
+                    <div class="card-content hide">
+                        <img class="first-image" src="/images/Img Placeholder.png" alt="" />
+                        <h2>Professional Art Printing Data</h2>
+                        <p>
+                            A daily selection of privately personalized reads; no accounts
+                            or sign-ups required. Has been the industry's standard.
+                        </p>
+                        <ul>
+                            <li>HTML</li>
+                            <li>Bootstrap</li>
+                            <li>Ruby</li>
+                        </ul>
+                    </div>
+                    <button class="btn-show">See project</button>
                 </div>
-                <button class="btn">See project</button>
-            </div>
 
-            <div class="card">
-                <div class="card-content">
-                    <h2>Profesional Art Printing Data</h2>
-                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the
-                        industry's standard.</p>
-                    <ul>
-                        <li>HTML</li>
-                        <li>Bootstrap</li>
-                        <li>Ruby</li>
-                    </ul>
+                <div class="card bg-2">
+                    <div class="card-content">
+                        <h2>Professional Art Printing Data</h2>
+                        <p>
+                            A daily selection of privately personalized reads; no accounts
+                            or sign-ups required. Has been the industry's standard.
+                        </p>
+                        <ul>
+                            <li>HTML</li>
+                            <li>Bootstrap</li>
+                            <li>Ruby</li>
+                        </ul>
+                    </div>
+                    <button class="btn">See project</button>
                 </div>
-                <button class="btn">See project</button>
-            </div>
```

Comment:
- [ ] kindly consider adding dark opacity on this card too on your desktop version so as to have an exact design as the one given  check the image below 👇 
<img width="774" alt="Screen Shot 2023-04-08 at 16 16 19" src="https://user-images.githubusercontent.com/60146030/230726246-e2f97326-601c-412a-bc6a-4c3bc59c62be.png">

Reply:
I've added dark opacity on this card